src/views/viewer/ChannelView/ChannelAbout.style.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from '@emotion/styled'
+
+import { Avatar, Text } from '@/shared/components'
+import { colors, media, sizes } from '@/shared/theme'
+
+export const TextContainer = styled.div`
+  display: grid;
+  grid-gap: ${sizes(4)};
+  padding-bottom: ${sizes(8)};
+  margin-bottom: ${sizes(8)};
+  border-bottom: 1px solid ${colors.gray[600]};
+`
+
+export const Container = styled.div`
+  display: grid;
+  gap: 142px;
+  grid-template-columns: minmax(0, 1fr) minmax(206px, 326px);
+  margin-bottom: 50px;
+  ${media.base} {
+    gap: 32px;
+    grid-template-columns: 1fr;
+    grid-template-rows: auto 1fr;
+  }
+  ${media.small} {
+    gap: 79px;
+    grid-template-rows: initial;
+    grid-template-columns: minmax(0, 1fr) minmax(206px, 228px);
+  }
+  ${media.medium} {
+    gap: 94px;
+    grid-template-columns: minmax(0, 1fr) minmax(206px, 260px);
+  }
```

Comment:
Where do those specific pixel values (142, 79, 206, etc) come from? I would expect our design to use some sensible values from theme for spacing, is that not the case?

Reply:
adjusted by a couple of pixels the ones that were not matching initially it works the same